.eslintrc.json

```diff
@@ -15,7 +15,6 @@
     },
     "rules": {
         "indent": ["error", 2],
-        "linebreak-style": ["error", "unix"],
```

Comment:
This should be reverted

Reply:
If we implement the gitattributes file, then the line endings will be converted to LF upon pushing to repository. I removed this line because I found it to be a redundant check in lint.

Do you think it's still worthwhile to have in?

Reply:
We'll need this to ensure that our lint is appropriate. Even if it may be redundant, without it we're not actually linting properly.